ovh/resource_dedicated_server_test.go

```diff
@@ -137,7 +136,7 @@ func TestAccDedicatedServer_basic(t *testing.T) {
 				ImportStateVerify:                    true,
 				ImportStateVerifyIdentifierAttribute: "service_name",
 				ImportStateVerifyIgnore: []string{
-					"display_name", "order", "ovh_subsidiary", "plan", "plan_option", "template_name",
+					"display_name", "order", "ovh_subsidiary", "plan", "plan_option", "operating_system",
```

Comment:
I guess this field could be filled with the `os` returned by the API when getting the server no ? Or maybe add this field on API side ?